opentrons-ai-client/src/resources/utils/createProtocolUtils.tsx

```diff
@@ -249,26 +251,58 @@ export function generateChatPrompt(
     args_0: CreatePrompt | ((prev: CreatePrompt) => CreatePrompt)
   ) => void
 ): string {
-  const defs = getOnlyLatestDefs()
-
   const robotType = t(values.instruments.robot)
   const scientificApplication = `- ${t(
     values.application.scientificApplication
   )}`
   const description = `- ${values.application.description}`
+
+  // we need to do this nonsense to convert pipette names to api load names
+  // this data does not yet live in  pipette defs, but hopefully aill within 6 months
+  // of writing this comment. https://opentrons.atlassian.net/browse/EXEC-1426
+  let leftPipetteApiLoadName: string | null = null
+  let rightPipetteApiLoadName: string | null = null
+
+  if (values.instruments.pipettes === TWO_PIPETTES) {
+    const leftPipetteSpecs = getPipetteSpecsV2(
+      values.instruments.leftPipette as PipetteName
+    )
+    const rightPipetteSpecs = getPipetteSpecsV2(
+      values.instruments.rightPipette as PipetteName
+    )
+    if (leftPipetteSpecs != null) {
+      leftPipetteApiLoadName = getFlexNameConversion(leftPipetteSpecs)
+    }
+
+    if (rightPipetteSpecs != null) {
+      rightPipetteApiLoadName = getFlexNameConversion(rightPipetteSpecs)
+    }
```

Comment:
I checked locally that this logic only worked when a user selects both pipettes. 
We may choose only one pipette sometimes. This means one of them  (`values.instruments.leftPipette` or `values.instruments.rightPipette`) is None. With None `getPipetteSpecsV2` breaks. 

How about changing like so:
```suggestion
    const leftPipetteName = values.instruments.leftPipette
    const rightPipetteName = values.instruments.rightPipette

    // Check if leftPipetteName is valid before calling getPipetteSpecsV2
    if (leftPipetteName && leftPipetteName !== NO_PIPETTES) {
      const leftPipetteSpecs = getPipetteSpecsV2(leftPipetteName as PipetteName)
      if (leftPipetteSpecs != null) {
        leftPipetteApiLoadName = getFlexNameConversion(leftPipetteSpecs)
      }
    }

    // Check if rightPipetteName is valid before calling getPipetteSpecsV2
    if (rightPipetteName && rightPipetteName !== NO_PIPETTES) {
      const rightPipetteSpecs = getPipetteSpecsV2(rightPipetteName as PipetteName)
      if (rightPipetteSpecs != null) {
        rightPipetteApiLoadName = getFlexNameConversion(rightPipetteSpecs)
      }
    }
```

Reply:
good catch, ill fix this